### ISSUE TITLE
Support GHC alpha releases

### ISSUE THREAD

**Issue by bergmark** [haskell-CI/haskell-ci#588]
It would be great if it was easy to add GHC alpha's to haskell-ci builds.

I got this working for haskell-src-meta by:
* Adding the GHC version to haskell-ci: https://github.com/bergmark/haskell-ci/commit/5495b1884519c0e878e98077324d95d7eb6a3559
* Hacking the following lines into [the generated workflow file](https://github.com/haskell-party/haskell-src-meta/pull/37/commits/f49a55cc7568f58ae83d3aefc1454a58f554981a) - this could of course be done inside haskell-ci as well
```
            "$HOME/.ghcup/bin/ghcup" upgrade
            "$HOME/.ghcup/bin/ghcup" config add-release-channel https://raw.githubusercontent.com/haskell/ghcup-metadata/master/ghcup-prereleases-0.0.7.yaml
```

**Comment by phadej** [comment]
Yes, we can add these in dev versions. I would prefer to not keep alpha versions around in released `haskell-ci` version though, as they have little value after proper GHC version is released.

**Comment by bergmark** [comment]
How should dev versions be managed? A separate branch in this repo?

**Comment by bergmark** [comment]
Wow, thanks @phadej!!
